lib/datura/to_es/csv_to_es/fields.rb

```diff
@@ -156,6 +159,10 @@ def source
     @row["source"]
   end
 
+  # nested field
+  def spatial
+  end
+
   def subjects
     if @row["subjects"]
       @row["subjects"].split("; ")
```

Comment:
It looks like you've got field names standardized with all plural field names using a delimiter / split.

These `# nested field` comments help denote those fields as different as well. I'm curious if we should do something with those method names to denote that so it's apparent when used in the code elsewhere when the comment won't be there for clarification. Something like `_n` at the end? Or maybe even just a hanging underscore? `spatial_n` or `spatial_`? :thinking:

Reply:
I think there's more standardization that we could do. A lot of the API fields are still holdovers from the Solr fields, which has been confusing to me as a programmer when I'm quickly trying to guess a field name!

I think it's reasonable to try to distinguish the nested fields by method name, although keep in mind that if we change that, they will no longer be identical to the API fields. In the above PR I had gone through and tried to get rid of all name discrepancies, so we would be reintroducing some if we did that to nested fields.